src/ua.c

```diff
@@ -81,6 +87,7 @@ static struct {
 #endif
 };
 
+static struct list *custom_hdrs;
```

Comment:
it is better to move this to `struct ua` so that it becomes re-entrant.

the long term goal of baresip is to make it 100% re-entrant with no static/global data.